test/project_types/extension/features/argo/base_test.rb

```diff
@@ -0,0 +1,253 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'project_types/extension/extension_test_helpers'
+require 'base64'
+require 'pathname'
+
+module Extension
+  module Features
+    module Argo
+      class BaseTest < MiniTest::Test
+        include TestHelpers::FakeUI
+        include ExtensionTestHelpers::Stubs::ArgoScript
+
+        def setup
+          super
+          ShopifyCli::ProjectType.load_type(:extension)
+
+          @dummy_argo = ExtensionTestHelpers::DummyArgoSetup.new
+          @result = 'fake result'
+          @error_message = "fake error"
+          @no_error = ''
+          @encoded_script = Base64.strict_encode64('var fake={}')
+        end
+
+        def test_config_aborts_with_error_if_script_file_doesnt_exist
+          ShopifyCli::JsSystem.any_instance.stubs(:package_manager).returns('yarn')
+          Argo::Base.any_instance.stubs(:run_yarn_install).returns(true)
+          Argo::Base.any_instance.stubs(:run_yarn_run_script).returns(true)
+          @dummy_argo.stubs(:extract_argo_renderer_version).returns('0.0.1')
```

Comment:
It may make the test clearer if you include the following: `File.stubs(:exist?).returns(false)`.